### PR TITLE
Optimize size of emoji list (stop using tiptap extension)

### PR DESCRIPTION
**Pull request by sipec**
Now the emoji list is only 51K

### CODE REVIEW

File: web/components/editor/emoji/emoji-extension.ts [hunk 1]
```diff
@@ -2,28 +2,19 @@ import { Extension } from '@tiptap/core'
...
-const emojis: EmojiItem[] = [
-  ...ttEmojis,
-  {
-    name: 'mana',
-    shortcodes: ['m$, mana'],
-    tags: ['money'],
-    emoji: ENV_CONFIG.moneyMoniker,
-  },
-]
+// {
```
Comment (akrolsmir) [inline web/components/editor/emoji/emoji-extension.ts:8]: Let's remove this unused code now

**Review by akrolsmir — APPROVED** [review]
Looks good, thanks! I'm a bit sad that things like :) won't automatically convert to 😄 anymore, but it's not that important (we can patch it in later, if we want)

**Comment by mqp** [comment]
Thanks for improving this, it was on my list of "stuff I wanted to deal with at some point."

File: web/components/editor/emoji/emoji-suggestion.ts [hunk 2]
```diff
@@ -21,26 +26,41 @@ const rank = (c: string) => {
   return r < 0 ? 101 : r
 }
 
+const emojiArr = Object.entries(invertBy(shortcodes)).map(
```
Comment (mqp) [inline web/components/editor/emoji/emoji-suggestion.ts:29]: Kind of odd to store the file as a `shortcode: codepoint` map if you are immediately going to transform it into a `codepoint: [shortcodes]` list. Why not just store it how you want it?

**Pull request merged**